### DISCUSSION TITLE
Help appreciated - Zone/Object Snapshots in Home Assistant

### DISCUSSION THREAD

**Discussion by dalewright** [blakeblackshear/frigate#9198] [Ask A Question] [answered]
Happy new year all,

I have been using Frigate for years and haven't quite worked out if it's possible to get object-based snapshots in Home  Assistant from a specific zone only.

For example:

When a car drives passes my gate but doesn't turn into my drive, the Frigate UI ignores the car (based on required_zones in my camera config) and does not create an event (amazing!)

However, home assistant integration pulls in a snapshot of the car passing via MQTT.

I would like to only get a snapshot in Home Assistant if the car "entered the zone" 

Any help would be appreciated.

**Comment by NickM-27** [comment, marked as answer]
This can be done by setting required zones for MQTT at the camera level

**Reply by dalewright** [reply]
I am embarrassed it was so simple/obvious! Cheers mate

**Reply by dalewright** [reply]
@NickM-27  Thoughts on adding this topic to the following docs?

https://docs.frigate.video/configuration/zones#restricting-events-to-specific-zones

**Reply by NickM-27** [reply]
probably makes sense, as a different section
